packages/flutter/lib/src/rendering/sliver.dart

```diff
@@ -522,6 +522,7 @@ class SliverGeometry extends Diagnosticable {
     double layoutExtent,
     this.maxPaintExtent = 0.0,
     this.maxScrollObstructionExtent = 0.0,
+    this.hitTestOrigin = 0.0,
```

Comment:
Should this maybe be initialized with the paintOrigin? Below it looks like that hitTestExtent assumes paintExtent if you don't set anything explicitly.